contribs/vsp/src/main/java/playground/vsp/scoring/IncomeDependentUtilityOfMoneyPersonScoringParameters.java

```diff
@@ -80,11 +77,11 @@ private double computeAvgIncome(Population population) {
 				"Income values <= 0 are ignored. Agents that have negative or 0 income will use the marginalUtilityOfMOney in their subpopulation's scoring params..");
 		OptionalDouble averageIncome =  population.getPersons().values().stream()
 				.filter(person -> person.getAttributes().getAttribute(PERSONAL_INCOME_ATTRIBUTE_NAME) != null) //consider only agents that have a specific income provided
-				.mapToDouble(person -> (double) person.getAttributes().getAttribute(PERSONAL_INCOME_ATTRIBUTE_NAME))
+				.mapToDouble(person -> Double.parseDouble((String) Objects.requireNonNull(person.getAttributes().getAttribute(PERSONAL_INCOME_ATTRIBUTE_NAME))))
```

Comment:
I think you do not need to check `Objects.requireNonNull` for the `PERSONAL_INCOME_ATTRIBUTE_NAME` attribute as one line above you are actually filtering out persons without this attribute.

Reply:
Yeah, it's suggested by the IntelliJ. I can remove that.